src/org/usfirst/frc/team2399/robot/OI.java

```diff
@@ -130,4 +96,24 @@ public OI(){
 		shiftToHotButt.whenPressed(shiftToHot);
 		shiftToDangerousButt.whenPressed(shiftToDangerous);
 	}
+	
+	public static double getLeftY() {
+		return leftJoy.getY() * RobotMap.JOYDRIVE_FORWARD;
+	}
+
+	public static double getRightY() {
+		return rightJoy.getY() * RobotMap.JOYDRIVE_FORWARD;
+	}
+	
+	public static double deadband(double input)
```

Comment:
I'd like this to have a better name. Quick craftsmanship lesson:
A **routine** is another term for what we've been calling **methods**. There are two types of routines:
1. **Functions**: Routines that compute and return a value.
2. **Procedures**: Routines that do something line-by-line.

In this case, we're dealing with a function, and functions are typically named based on what they return. For example, `Math.abs()` is so named because it computes the absolute value of its argument.

My question for you: This method is called `deadband`. Does it compute the deadband? If not, what might be a better name? _Hint:_ These are often named in the format verbNoun.

_Note:_ Also make sure you avoid generic names, e.g. `do[Something]` or `manage[Thing]`.